bin/dhttpd.dart

```diff
@@ -1,3 +1,40 @@
-import '_boot.dart';
+import 'dart:io';
+import 'package:args/args.dart';
+import 'package:dhttpd/dhttpd.dart';
 
-main(List<String> args) => bootServer(args);
+main(List<String> args) async {
+  var argParser = new ArgParser()
+    ..addOption('port',
+        abbr: 'p',
+        defaultsTo: DEFAULT_PORT.toString(),
+        valueHelp: 'port',
+        help: 'The port to listen on.')
+    ..addOption('path',
+        valueHelp: 'path', help: 'The path to serve (defaults to the cwd).')
+    ..addOption('host',
+        defaultsTo: DEFAULT_HOST.toString(),
+        valueHelp: 'host',
+        help: 'The hostname to listen on (defaults to "localhost").')
+    ..addFlag('help', abbr: 'h', negatable: false, help: 'Displays the help.');
+
+  var results = argParser.parse(args);
+
+  if (results['help']) {
+    print(argParser.usage);
+    exit(1);
```

Comment:
[nit] I usually expect asking for help to not be considered a failure

Reply:
ack...